miso-web/src/main/resources/internal/miso.properties

```diff
@@ -27,6 +27,9 @@ miso.baseDirectory:/storage/miso/
 miso.fileStorageDirectory:/storage/miso/files/
 miso.submissionStorageDirectory:/storage/miso/files/submission/
 
+# Where to send bug reports for your organisation
+miso.bugUrl:https://github.com/TGAC/miso-lims/issues/new
```

Comment:
Might be worth keeping it in both? I'm not sure what distinctions EI makes between internal and external, or why internal exists since all the documentation specifies external.